crates/bevy_pbr/src/light_probe/environment_map.wgsl

```diff
@@ -0,0 +1,121 @@
+#define_import_path bevy_pbr::environment_map
+
+#import bevy_pbr::mesh_view_bindings as bindings
+#import bevy_pbr::mesh_view_bindings::light_probes
+
+struct EnvironmentMapLight {
+    diffuse: vec3<f32>,
+    specular: vec3<f32>,
+};
+
+fn environment_map_light(
+    perceptual_roughness: f32,
+    roughness: f32,
+    diffuse_color: vec3<f32>,
+    NdotV: f32,
+    f_ab: vec2<f32>,
+    N: vec3<f32>,
+    R: vec3<f32>,
+    F0: vec3<f32>,
+    world_position: vec3<f32>,
+) -> EnvironmentMapLight {
+    var out: EnvironmentMapLight;
+
+    // Split-sum approximation for image based lighting: https://cdn2.unrealengine.com/Resources/files/2013SiggraphPresentationsNotes-26915738.pdf
+    // Technically we could use textureNumLevels(environment_map_specular) - 1 here, but we use a uniform
+    // because textureNumLevels() does not work on WebGL2
+    let radiance_level = perceptual_roughness * f32(bindings::lights.environment_map_smallest_specular_mip_level);
+
+#ifdef ENVIRONMENT_MAP_LIGHT_PROBES
```

Comment:
I think we should rename this shader def to `MULTIPLE_LIGHT_PROBES_IN_ARRAY` or something like that.

Reply:
Done